Specs/Scene/MetadataTablePropertySpec.js

```diff
@@ -484,42 +495,46 @@ describe("Scene/MetadataTableProperty", function () {
   it("get returns fixed size arrays", function () {
```

Comment:
Add tests for getting/setting an array of vectors/matrices (once the array packing is available)